qcengine/programs/util/hessparse.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
```

Comment:
Can we just have a single util file? I always found the multiple files under util to be troublesome unless the utils becomes truly huge.

Reply:
Huh, I find the mixed-bag files like `qcengine/util.py` to be troublesome. A dictionary subclass and a Hessian reader just don't sound like they belong together. I expect some general options fns next. Can we revisit this when Cfour and Gamess interfaces are ready to register? There's ppl who'll be working on this, and it'd be nice to have files just migrate rather than migrate and rechristen.

Reply:
Sounds good.